packages/thirdweb/src/wallets/smart/lib/bundler.ts

```diff
@@ -144,7 +145,7 @@ export async function getUserOpReceipt(
   if (!res) {
     return undefined;
   }
-  if (res.success === false) {
+  if (res.success === false && args.decodeRevertReason) {
```

Comment:
this changes default behavior, not sure if we can do this

instead what could work is adding a `skipDecodeRevertReason` (inverse logic) so the default behavior stays the same but you can explicitly skip this step

Reply:
makes sense. will do

Reply:
@jnsdls made the updates